cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -373,23 +373,21 @@ def publication_additional(self, key, value):
     empty = not bool(_publication_info)
     for i, v in enumerate(force_list(value)):
         temp_info = {}
-        pages = clean_pages_range("k", v)
+        pages = clean_val("k", v, str)
         if pages:
-            temp_info.update(pages)
+            temp_info.update(pages=pages)
         rel_recid = clean_val("b", v, str)
         if rel_recid:
             _related.append(
                 {
                     "related_recid": rel_recid,
                     "relation_type": OTHER_RELATION.name,
-                    "relation_description": "chapter of"
+                    "relation_description": "is chapter of the book"
```

Comment:
Is it ok to put the word `book` and not literature? is it always a book the parent?

Reply:
i think we could leave it as `is chapter of` to make it generic for all document types

Reply:
I changed it as requested in the [issue](https://github.com/CERNDocumentServer/cds-migrator-kit/issues/75). ( description: "is chapter of the book" )

Reply:
OK I think we should discuss this on the standup